File: in-motion-server/Post/IMS.Post.Models/Dto/Outgoing/PostCommentDto.cs
namespace IMS.Post.Models.Dto.Outgoing;

public class PostCommentDto
{
    public string Id { get; set; }
    public PostAuthorDto Author { get; set; }
    public string Content { get; set; }
    public string PostId { get; set; }
    public DateTime CreatedAt { get; set; }
    
}

File: in-motion-android/app/src/main/java/com/inmotion/in_motion_android/data/remote/dto/posts/comments/PostCommentDto.kt
package com.inmotion.in_motion_android.data.remote.dto.posts.comments

import com.inmotion.in_motion_android.data.remote.dto.posts.PostAuthorDto

data class PostCommentDto(
    val id: String,
    val author: PostAuthorDto,
    val content: String,
    val postId: String,
    val postCommentReactionCount: String,
    val createdAt: String
)
